tethysapp/gwdm/public/js/update_aquifer.js

```diff
@@ -79,15 +79,15 @@ var LIBRARY_OBJECT = (function() {
     });
 
     var viewIcon = function(cell, formatterParams){ //plain text value
-        return "<span class='glyphicon glyphicon-sunglasses view-aquifer-tabulator'></span>";
+        return "<span class='sunglasses view-aquifer-tabulator'></span>";
```

Comment:
Need to change `<span>` to `<i>` and add `bi` to classes:

e.g.:
```
return "<i class='bi bi-sunglasses view-aquifer-tabulator'></i>";
```

Fix all instances of this in this file.